src/Alpha_complex/include/gudhi/Alpha_complex.h

```diff
@@ -248,7 +261,8 @@ class Alpha_complex {
   template <typename SimplicialComplexForAlpha,
             typename Filtration_value = typename SimplicialComplexForAlpha::Filtration_value>
   bool create_complex(SimplicialComplexForAlpha& complex,
-                      Filtration_value max_alpha_square = std::numeric_limits<Filtration_value>::infinity()) {
+                      Filtration_value max_alpha_square = std::numeric_limits<Filtration_value>::infinity(),
+                      bool exact = false) {
```

Comment:
I was wondering if we should try to write this extra argument in a way that will naturally extend if we ever add a safe2 mode. But I guess nobody will use exact anyway, so we can slightly break the interface when the time comes if needed.